src/shared/ui/surfaces/Card/Card.tsx

```diff
@@ -34,22 +51,32 @@ export function Card({ roundedPicture }: CardProps) {
         <IconButton
           className="absolute bottom-spacing-tighter-2 right-spacing-tighter-2 translate-y-spacing-tighter-2 opacity-0 duration-300 group-hover:translate-y-0 group-hover:opacity-100"
           size="large"
-          tooltip="Play Todays Top Hits"
+          tooltip={`Read: ${tooltipInfo}`}
           variant="accent"
         >
           <HiPlay />
         </IconButton>
       </div>
       <div className="mt-spacing-base flex flex-col">
-        <span className="font-bold text-color-base">Todays Top Hits</span>
-        <Link
-          href="#"
-          className="line-clamp-2 text-font-size-smaller font-semibold"
-          underline="hover"
-          variant="muted"
-        >
-          Tate McRae is on top of the Hottest 50!
-        </Link>
+        <span className="truncate font-bold text-color-base">{title}</span>
+        <div className="flex items-center gap-spacing-tighter-4">
+          <Link
+            href="#"
+            className="line-clamp-2 text-font-size-smaller font-semibold"
+            underline="hover"
+            variant="muted"
+          >
+            {subtitle}
+          </Link>
+          {subtitleBis && (
+            <>
+              <p className="text-color-subdued">•</p>
+              <p className="truncate text-font-size-smaller text-color-subdued">
+                {subtitleBis}
+              </p>
+            </>
+          )}
+        </div>
```

Comment:
We'll discuss another way to implement this pattern :)

Reply:
See together, we will implement it later !